synapse/replication/http/send_event.py

```diff
@@ -46,6 +46,12 @@ class ReplicationSendEventRestServlet(ReplicationEndpoint):
             "ratelimit": true,
             "extra_users": [],
         }
+
+        200 OK
+
+        { "stream_id": ..., "event_id": ... }
```

Comment:
to show that stream_id is an int: 

```suggestion
        { "stream_id": 12345, "event_id": "$abcdef..." }
```